src/utilities/_hypre_utilities.h

```diff
@@ -1599,10 +1610,10 @@ HYPRE_Real    hypre_cimag( HYPRE_Complex value );
 HYPRE_Complex hypre_csqrt( HYPRE_Complex value );
 #else
 #define hypre_conj(value)  value
-#define hypre_cabs(value)  fabs(value)
+#define hypre_cabs(value)  hypre_abs(value)
 #define hypre_creal(value) value
 #define hypre_cimag(value) 0.0
-#define hypre_csqrt(value) sqrt(value)
+#define hypre_csqrt(value) hypre_sqrt(value)
```

Comment:
How did `hypre_csqrt` work before when using "single" or "quad"?  Maybe we don't even use this function anywhere yet?

Reply:
@falgout the current HYPRE_COMPLEX is defined to only support double precision, so it would be fine if hypre is built with --enable-complex. However, without this build option, I would expect there to have been a warning, since sqrt is only for double precision. Using hypre_sqrt should fix that issue. I see hypre_csqrt being used in par_fsai_setup.c.

Reply:
Good point that you would expect to get a warning when `par_fsai_setup.c` was compiled with "single" or "quad" turned on.  It's not just a question of testing.  Hmmm...  In any case, we need the functionality of this `hypre_sqrt` macro to handle all of the real cases, so this is okay.